Data/ByteString/Lazy.hs

```diff
@@ -41,6 +41,11 @@
 --
 -- > import qualified Data.ByteString.Lazy as B
 --
+-- Note: Using lazy I\/O functions like 'readFile' or 'hGetContents' largely
+-- gives up on prompt resource management. The underlying files or sockets
+-- will either be closed when EOF is reached, or when the underlying 'Handle'
+-- becomes unreachable and is collected by the garbage collector.
```

Comment:
I think this note is not very likely to be noticed in this position. If a general note like this is necessary, it should probably be in the `IO with ByteStrings` section.